src/app/features/tasks/task/task.component.ts

```diff
@@ -218,6 +221,15 @@ export class TaskComponent implements OnInit, OnDestroy, AfterViewInit {
       setTimeout(() => {
         this.focusTitleForEdit();
       });
+    } else {
+      this._route.queryParams.subscribe((params) => {
```

Comment:
This needs an unsubscribe :)

Reply:
Since I'm rather new to Angular/RxJS I looked it up and found [this](https://stackoverflow.com/questions/41138081/do-i-have-to-unsubscribe-from-activatedroute-e-g-params-observables), saying that angular takes care of this specific subscription. The thing is, I actually really don't like that. I believe it's incredibly hard to remember all these rules and explicitly unsubscribing would be clearer and possibly safer... Will happily do so, if you prefer...